beheer.md

```diff
@@ -39,48 +39,54 @@ Beheer helpt API developers/designers door:
+  * Dit kan door het steekproefsgewijs testen van Openapi.yaml vs test omgeving, vergelijk de resource.
 * De standaard is formeel goedgekeurd, concreet gemaakt.
-_Het forum voor standaardisatie stelt als eis dat er een openbare consiltatie is geweest. De vraag is alleen hoe die openbare consulatatie moet worden ingericht. Volgende acties zouden i.i.g. moeten zijn genomen:_
+_Het forum voor standaardisatie stelt als eis dat er een openbare consultatie is geweest. De vraag is alleen hoe die openbare consultatie moet worden ingericht. Volgende acties zouden i.i.g. moeten zijn genomen:_
   * Er moet minstens één API-lab gehouden zijn van deze API om de ervaringen van de gebruikers te verzamelen.
-  * Er moeten één of meer tussentijdse (sprint) demo's zijn geweest van deze API
+  * Er moeten één of meer tussentijdse (sprint) demo's zijn geweest van deze API.
   * ...
-* De OAS3 en de RI sluiten naadloos op elkaar aan.
-  * Dit kan door het steekproefsgewijs testen van Openapi.yaml vs test omgeving, vergelijk de resource
-* De testomgeving met een implementatie van de RI moet draaien. Dit betekent ook dat:
-  * De testscripts voor de RI waarmee we al het gewenst gedrag zoals beschreven in standaard testen ook beschikbaar zijn gesteld. Beheer zal dit bij doorontwikkeling weer nodig hebben en tevens zullen zij deze beschikbaar stellen aan de leveranciers/gemeenten. 
-* De docker container met de RI moet beschikbaar zijn op DockerHub in de VNG namespace.
-* Documentatie omvat minimaal:
+* De docker container met de RI moet beschikbaar zijn op DockerHub in de VNG namespace;
+* De documentatie omvat minimaal:
   * Een UML model van het informatiemodel (de objecttypen en onderlinge relaties) zoals deze in de API gebruikt wordt.
   * Een overzicht van welke gegevens worden ontsloten in de vorm van een tabel van de resource + bijzonderheden. 
-  * Installatie instructies van de referentie implementatie (bijv. INSTALL.rst in de repository).
-  * Een beschrijving van de business logica die in de standaard van toepassing is.
-  * Een overzicht van wijzigingen (bijv. CHANGELOG.rst in de repository) met daarin per versie wat is toegevoegd, gewijzigd en hoe te migreren van de voorgaande naar deze versie.
-  * De standaard bevat een overzicht waarin wordt aangegeven welke versies van de betreffende API standaard met welke versies van andere API standaarden compatible zijn.
- * De standaard bevat een overzicht van alle partijen en personen die bij de ontwikkeling van de standaard betrokken zijn (geweest).
+  * Installatie instructies van de RI (bijv. INSTALL.rst in de repository).
+* De testomgeving met een implementatie van de RI moet probleemloos draaien. Dit betekent ook dat:
+  * De testscripts voor de RI waarmee we al het gewenst gedrag zoals beschreven in standaard testen ook beschikbaar zijn gesteld. Beheer zal dit bij doorontwikkeling weer nodig hebben en tevens zullen zij deze beschikbaar stellen aan de leveranciers/gemeenten. 
```

Comment:
De testscripts voor de RI waarmee we al het gewenst gedrag zoals beschreven in standaard testen ook beschikbaar zijn gesteld 
Die zin loopt niet lekker. Misschien beter 
De testscripts voor de RI waarmee al het gewenst gedrag zoals beschreven in standaard testen ook beschikbaar wordt gesteld
?

Reply:
Is aangepast.